src/components/Home/BookList/BookList.js

```diff
@@ -1,17 +1,25 @@
+import { useEffect } from 'react';
 import './BookList.css';
-import { useSelector } from 'react-redux';
+// import axios from 'axios';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchBooks } from '../../../redux/books/booksSlice';
 import Book from './Book';
 
 function BookList() {
-  const { books } = useSelector((state) => state.books);
+  const { books, error, isLoading } = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, [dispatch]);
 
   return (
     <ul className="book-list">
-      {
-        books.map((book) => (
-          <Book book={book} key={book.id} />
-        ))
-      }
+      {isLoading && <p>Loading</p>}
+      {error && <p>Error</p>}
+      {books && books.map((book) => (
+        <Book book={book} key={book.item_id} />
+      ))}
     </ul>
   );
 }
```

Comment:
- [ ] `README.md` file: As required for all projects and to keep your documentation be professional, kindly make sure you fix your broken link of your live demo as it shows a `404` error page once it was cliked. To help you how to deploy a react app, you refer this **[link](https://www.youtube.com/watch?v=IlZPILaACGw)**.
![Screenshot from 2023-04-06 20-14-47](https://user-images.githubusercontent.com/83042742/230515633-4aff033c-706e-498f-8ee1-4dbf8eb9667f.png)